### PR TITLE
Adjustable profile update.

### PR DESCRIPTION
**Pull request by sodanakin**
**Fixed issue:** [Reference the issue number here, or remove if not a fix]

#457 (my own request lol)

**Changes made:**

I split the Hatcheggs and profile loop into two loops whereby I added control over the timer of the profile loop. Which you can now adjust in the config.
Updated the settings.kt file accordingly.

### CODE REVIEW

File: config.properties.template [hunk 1]
```diff
@@ -85,6 +85,9 @@ always_curve=false
 # Never use berries
 never_use_berries=true
 
+# Set Timer for profile,Xp,lvl update. (Default: 60)
+set_timer=60
```
Comment (jabbink) [inline config.properties.template]: Please rename to profile_update_timer

File: src/main/kotlin/ink/abb/pogo/scraper/Settings.kt [hunk 2]
```diff
@@ -18,6 +18,8 @@ import java.util.*
 
 class Settings(val properties: Properties) {
 
+	val profileUpdateTimer = getPropertyIfSet("Set Timer", "set_timer", 60, String::toLong)
```
Comment (jabbink) [inline src/main/kotlin/ink/abb/pogo/scraper/Settings.kt]: if you do the fix from above, edit it here as well

File: src/main/kotlin/ink/abb/pogo/scraper/Bot.kt [hunk 3]
```diff
@@ -98,8 +98,11 @@ class Bot(val api: PokemonGo, val settings: Settings) {
         runningLatch = CountDownLatch(1)
         phaser = Phaser(1)
 
-        runLoop(TimeUnit.SECONDS.toMillis(60), "ProfileLoop") {
+        runLoop(TimeUnit.SECONDS.toMillis(settings.profileUpdateTimer), "ProfileLoop") {
             task(profile)
+        }
+		
+	runLoop(TimeUnit.SECONDS.toMillis(60), "HatchEggsLoop") {
             task(hatchEggs)
```
Comment (jabbink) [inline src/main/kotlin/ink/abb/pogo/scraper/Bot.kt]: not sure if hatchEggs relies on the profile update (it does need a fresh inventory). If that's the case, please remove this runLoop and put the eggs behind the task(profile) again

**Comment by sodanakin** [comment]
Can't test atm PTC is down I will edit it in a bit when servers aren't puking rainbows :")
Also about the HatchEggs I had the bot running with this implemented since I added it without issues. So do I need to put it back in the same loop or not? Just want to be sure what to edit before making a new pull request. @jabbink

**Pull request merged**